code/modules/uplink/uplink_item.dm

```diff
@@ -385,7 +386,6 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "An alternative 8-round stun slug magazine for use with the Bulldog shotgun. \
 			Saying that they're completely non-lethal would be lying."
 	item = /obj/item/ammo_box/magazine/m12g/stun
-	cost = 3
```

Comment:
it's not actually parent, it's two types with the same path
just rename one of them please